Source/Fuse.Maps/MapView.uno

```diff
@@ -219,6 +240,16 @@ namespace Fuse.Controls
 			UpdateMarkersNextFrame();
 		}
 
+		void OnPolylineAdded(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
+
+		void OnPolylineRemoved(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
```

Comment:
I think this cleanup should be done together with fixing `OnMarkerAdded`/`OnMarkerRemoved`.